dpnp/backend/include/dpnp_iface.hpp

```diff
@@ -229,10 +229,11 @@ INP_DLLEXPORT void dpnp_sort_c(void* array, void* result, size_t size);
  *
  * @param [in]  array   Input array with data.
  * @param [out] result  Output array.
- * @param [in]  shape   Shape of input array.
+ * @param [in]  size    Number of elements in input arrays.
+ * @param [in]  size_   Number of elements in each parts input arrays.
  */
 template <typename _DataType>
-INP_DLLEXPORT void dpnp_cholesky_c(void* array1_in, void* result1, size_t* shape);
+INP_DLLEXPORT void dpnp_cholesky_c(void* array1_in, void* result1, size_t size, size_t size_);
```

Comment:
Please make variables names more understandable. 
These are look like a shape of input array, but not sure in it